lib/launcher.rb

```diff
@@ -132,9 +132,7 @@ def launch
       # do nothing. This is to prevent the exception handler below from catching SystemExit exceptions.
       raise
     rescue Exception => e
-      logger.error("Error during startup, terminating. #{e}")
-      logger.debug("#{e}:#{e.backtrace.join("\n")}")
-
+      logger.error("Error during startup, terminating. #{e}", e.backtrace)
```

Comment:
this was split on purpose, in order not to pollute error-level log with backtraces.